### ISSUE TITLE
[Ethernet] Circular dependency in Python

### ISSUE THREAD

**Issue by SkypLabs** [kaitai-io/kaitai_struct_formats#235]
Hi,

The Python interpreter fails to import the Ethernet parser present in the format gallery as you can see in the following stack trace:

```
Traceback (most recent call last):
  File "arp_parser.py", line 1, in <module>
    from ethernet_frame import EthernetFrame
  File "/home/user/Downloads/kaitai-struct-compiler-0.8/ethernet_frame.py", line 11, in <module>
    from ipv4_packet import Ipv4Packet
  File "/home/user/Downloads/kaitai-struct-compiler-0.8/ipv4_packet.py", line 10, in <module>
    from protocol_body import ProtocolBody
  File "/home/user/Downloads/kaitai-struct-compiler-0.8/protocol_body.py", line 14, in <module>
    from ipv4_packet import Ipv4Packet
ImportError: cannot import name 'Ipv4Packet'
```

This is caused by a circular dependency issue.

Python version: 3.5.3
Format gallery commit id: 78ec8420f7da1d6a2a1a6f8271558765ebd646f9

**Comment by dgelessus** [comment]
This issue has been fixed in version 0.9 of kaitai-struct-compiler, see kaitai-io/kaitai_struct#337. Kaitai Struct 0.9 is not released yet, so to use it you need to download the development/unstable build of KS, or compile it yourself from source.

**Comment by SkypLabs** [comment]
Hi @dgelessus,

Thanks for your reply. Sorry for the duplicate. I didn't find the issue that you mentioned. I should have searched better.

This issue can be closed.
